src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,30 @@
+.dg_summary-list {
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: $padding-largest;
+  text-decoration: none;
+  p {
+    color: $gray;
+    font-family: $default-heading-font;
+    font-size: $smallest-heading-font-size;
+    line-height: 30px;
+  }
+  a {
+    text-decoration: none;
+    background-color: transparent;
+  }
+  .bg-light {
+    background-color: $gray-lightest;
+  }
+
+  h3 {
+    color: $gray;
+  }
+
+  .side-list-ico {
+    font-size: 30px;
+    color: $gray;
+    text-align: center;
+    width: 30px;
```

Comment:
Just a note to remember, I think this will probably change slightly after the flexbox changes are made.

Reply:
done